src/reference/asciidoc/transactions.adoc

```diff
@@ -164,6 +164,10 @@ In each case, the _#root_ variable is the original `Message`; in some cases, oth
 For example, the `MongoDbMessageSource` provides the _#mongoTemplate_ variable which references the message source's `MongoTemplate`; the `RedisStoreMessageSource` provides the _#store_ variable which references the `RedisStore` created by the poll.
 
 To enable the feature for a particular poller, you provide a reference to the `TransactionSynchronizationFactory` on the poller's <transactional/> element using the _synchronization-factory_ attribute.
+Spring Integration 5.0 introduces `PassthroughTransactionSynchronizationFactory` which is applied for pollers if there
+is no `TransactionSynchronizationFactory` available and the TX Advice is an instance of `TransactionInterceptor`.
+A `PassthroughTransactionSynchronizationFactory` can also be explicitly configured if you are using a a custom TX
```

Comment:
Double `a` and it really must be a separate paragraph.

Plus it would be helpful to say that now `AbstractPollingEndpoint` correctly throws `MessagingException` with the polled message as `failedMessage` property. And say that it is picked up from the `TransactionSynchronizationManager`.
Having a note about `PassthroughTransactionSynchronizationFactory` confuses me a bit now. 
The question is: what is the purpose of this factory used by default?

Let me know if I'm not clear yet and I'll come back to you with the polishing as I see for review and other comments.